### PR TITLE
feat: add rss feed for news

### PR DESCRIPTION
**Pull request by nurRiyad**
- Generate rss feed for only news contents
- Rss feed user `/news/feed.xml`
- Use `rss` npm package with `@types/rss` for type support

### CODE REVIEW

**Comment by romangg** [comment]
Generating the website with `pnpm generate` doesn't create the path.

**Comment by romangg** [comment]
You could use the rss package like here: https://github.com/subdiff/subdiff.org/blob/master/server/routes/blog/feed.xml.ts

**Comment by romangg** [comment]
Thanks. Can the RSS feed go into `manjaro.org/news/feed.xml`.

**Comment by nurRiyad** [comment]
> Thanks. Can the RSS feed go into `manjaro.org/news/feed.xml`.

Sure updating

File: server/routes/news/feed.xml.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import RSS from 'rss'
+import { serverQueryContent } from '#content/server'
+
+export default defineEventHandler(async (event) => {
+  const feed = new RSS({
+    title: 'Manjaro Linux Official',
+    description: 'This page provides an overview of Manjaro Linux, an open source operating system designed for ease of use. Learn about its features, installation, and support options. Get the most out of Manjaro Linux with the latest news, downloads, and tips from our helpful community',
+    site_url: 'https://manjaro.org',
+    feed_url: `https://manjaro.org/feed.xml`,
```
[the remark this answers is not in the document]
Comment (nurRiyad) [reply server/routes/news/feed.xml.ts:9]: Ohhh yes my mistake 😢

**Comment by nurRiyad** [comment]
@romangg  Review changes updated

**Comment by romangg** [comment]
Thank you! Working great. :)

**Comment by romangg** [comment]
It would be cool to have a small button/icon in https://manjaro.org/news to link to the RSS feed. Something like 

Screenshot_20241108_191043

at the bottom of https://subdiff.org/blog.

**Comment by nurRiyad** [comment]
> It would be cool to have a small button/icon in https://manjaro.org/news to link to the RSS feed. Something like
> 
> Screenshot_20241108_191043
> 
> at the bottom of https://subdiff.org/blog.

Ok updating.

**Comment by romangg** [comment]
Ah, I already merged this PR since the RSS feed itself can already go in. :)

Please open a new PR.

**Pull request merged**